examples/hello/BUILD.bazel

```diff
@@ -0,0 +1,65 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+load("//tools:cdk8s.bzl", "cdk8s_import", "cdk8s_synth")
+load("//tools:defaults.bzl", "jest_test", "ts_library")
+
+cdk8s_import(
+    name = "gen",
+    path = "examples/hello/imports",
+    language = "typescript",
+)
+
+ts_library(
+    name = "lib",
+    srcs = glob(
+        ["**/*.ts"],
+        exclude = ["**/*.test.ts"],
+    ),
+    deps = [
+        "//packages/cdk8s:lib",
+        "@npm//@aws-cdk/core",
+    ],
+)
+
+filegroup(
+    name = "sources",
```

Comment:
feels like this should be reused in the `ts_library` invocation above, no? I wouldn't want to specify sources twice

Reply:
The name sources here might be a little misleading. In this case what this means is the "compiled JS source" _from_ the `ts_library` rule. I can add a comment if this is still unclear.